src/app/components/attestation/attestation-create.jsx

```diff
@@ -135,159 +171,227 @@ function ChplAttestationCreate(props) {
 
   return (
     <ThemeProvider theme={theme}>
-      <Typography variant="h1">
-        Submit Attestation
-      </Typography>
+      <div className={classes.listingInspectHeader}>
```

Comment:
I think only one of the four styles on this div actually have an effect, and that's the padding one. Do we need this DIV when the padding could just be on a style attached to the Typography?